packages/commons/src/rate-limiter/rateLimiterUtils.ts

```diff
@@ -0,0 +1,15 @@
+import { OutgoingHttpHeaders } from "http2";
+import { RateLimiterStatus } from "./rateLimiterModel.js";
+
+export const rateLimiterHeadersFromStatus = (
+  rateLimiterStatus: RateLimiterStatus
+): OutgoingHttpHeaders => ({
+  "X-RateLimit-Limit": rateLimiterStatus.maxRequests,
+  "X-RateLimit-Interval": rateLimiterStatus.rateInterval,
+  "X-RateLimit-Remaining": rateLimiterStatus.remainingRequests,
+});
+// TODO why are these different from the ones in the openapi spec?
```

Comment:
IMHO: these are two different conventions, I would align the code to the openapi spec if possible